assets/src/hooks/useNearestIntersection.ts

```diff
@@ -1,38 +1,48 @@
-import { useEffect, useState } from "react"
+import { useCallback } from "react"
 import { fetchNearestIntersection } from "../api"
-import { FetchResult, loading } from "../util/fetchResult"
+import { FetchResult, loading, ok, fetchError } from "../util/fetchResult"
+import { useApiCall } from "./useApiCall"
 
+/**
+ * @deprecated use {@linkcode useNearestIntersection2}
+ *
+ * [⚙️ Refactor `useNearestIntersection` away from `FetchResult`](https://app.asana.com/0/1148853526253420/1207213296687090/f)
```

Comment:
Okay this is admittedly a kind of petty thing, but it seems like having this emoji in the file is causing Emacs to get confused about how long the line actually is, which totally trashes my development environment and makes it impossible for me to reliably edit this file. Would it be okay to remove it?

```suggestion
 * [Refactor `useNearestIntersection` away from `FetchResult`](https://app.asana.com/0/1148853526253420/1207213296687090/f)
```

Reply:
This makes me really sad [that your editor can't handle UTF8?...], but sure

Reply:
Honestly it might be iTerm. I dunno! I just know that _something_ started getting very glitchy, and when I deleted the emoji (and then closed emacs and re-opened it), it stopped glitching! 😅